helpers/blaze/range.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+
+function _range(from: number, to: number, range: number[] = []): number[] {
+  if (from === to) return [...range, from]
+  if (from > to) return _range(from - 1, to, [...range, from])
+  if (from < to) return _range(from + 1, to, [...range, from])
+
+  return range
+}
```

Comment:
I think it might be problematic to build ranges recursively, on my machine `_range(0, 5248)` already crashes.

Reply:
Thanks, I replaced it with a loop instead. Should be much faster now.